src/App.tsx

```diff
@@ -1,25 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
```

Comment:
This code is generating a list of items outside of the component. This is a bad practice because it's outside of the React lifecycle. It would be better to move this inside the component and use the `useState` hook to manage its state.